src/VisualStudio/Core/Def/Implementation/AnalyzerDependencyCheckingService.cs

```diff
@@ -46,7 +46,7 @@ internal sealed class AnalyzerDependencyCheckingService
             {
                 results = await GetConflictsAsync().ConfigureAwait(continueOnCapturedContext: true);
             }
-            catch (OperationCanceledException)
+            catch
```

Comment:
:exclamation: If this code references **Microsoft.VisualStudio.Shell.xx.0**, I think the handler should be changed to the following:

```csharp
catch (Exception ex)
{
  if (ErrorHandler.IsCritical(ex))
    throw;

  return;
}
```